### ISSUE TITLE
Launcher settings hidden in the system settings app

### ISSUE THREAD

**Issue by kugiigi** [ubports/ubuntu-touch#690]
- Device: Nexus 5
- Channel: xenial dev
- Build: 20180619-2

### Steps to reproduce
1. open the system settings app.
2. Look for the Launcher icon.
3. Alternatively, search for the word "Launcher".

### Expected behavior
The Launcher icon is displayed in the settings app.

### Actual behavior
The icon is missing in the settings app.
When searched, it appears to be listed in the result but the icon is still hidden.

**Comment by Flohack74** [comment]
Please clarifiy: I am using only German versions, and I compared 15.04 system settings to 16.04 - I do not see a missing icon, nor searching for Launcher or Starter (German similar word) Does bring up anything.

**Comment by kugiigi** [comment]
Of course, you won't see the missing icon because it's missing..just kidding :grin:
I think on tablets, this icon is always shown.

Here's a screenshot on vivid.

![screenshot20180620_003315758](https://user-images.githubusercontent.com/29456786/41641151-9bc9795e-7464-11e8-86f3-d65188f14831.png)

Here's a screenshot on xenial (Nexus 5 grid unit set to 10)
![screenshot20180620_170720872](https://user-images.githubusercontent.com/29456786/41648841-5029c6fa-747a-11e8-96b1-7bf6c28b2baf.png)

**Comment by Flohack74** [comment]
Thats smth to do with convergence maybe. We will have to find out if this is intentional or not, but probably it is since the mobile mode probably does not allow those 2 parameters to be customized.

**Comment by kugiigi** [comment]
Yes, I think it's intentional but the icon size setting I think is also applicable to staged mode/phones.

**Comment by kugiigi** [comment]
I just want to reopen the discussion about this because I really think this Launcher option should be available to users. Currently, both settings are available via UTTT but of course it would be better to make them available to users out of the box. Icon size is definitely applicable and make sense on any for factor. Always showing the launcher is only applicable to windowed mode but putting a simple note there I think would be enough to make it visible to everyone.

Currently in Nexus 7, the Launcher option isn't also available even though the indicator displays the desktop mode toggle. The Launcher option only shows when setting the grid unit to 10.

**Comment by mateosalta** [comment]
i'm sure we could put a mode check on the always show switch

**Comment by kugiigi** [comment]
There is already an open PR for this ubports/system-settings#269
Basically, it makes the `Launcher` page always visible while only showing the `Always show launcher` option when there's a large screen detected.
